src/constants.ts

```diff
@@ -100,10 +102,13 @@ export const FEDORA_RELEASES = new Map([
   [FEDORA_40, 'Fedora Linux 40'],
   [FEDORA_41, 'Fedora Linux 41'],
   [CENTOS_9, 'CentOS Stream 9'],
+  [CENTOS_10, 'CentOS Stream 10'],
 ]);
 
+export const RHEL_10_FULL_SUPPORT = ['2025-05-13', '2027-05-31'];
```

Comment:
Code looks good, does the full support end the same day the it will end for RHEL9?

Reply:
oh no
sorry that's meant to be 2030